packages/amplication-server/src/services/segmentAnalytics/segmentAnalytics.service.ts

```diff
@@ -34,6 +34,11 @@ export enum EnumEventType {
   CodeGeneratorVersionUpdate = "codeGeneratorVersionUpdate",
 
   RedeemCoupon = "RedeemCoupon",
+
+  // break the monolith
+  StartRedesign = "startRedesign",
+  BreakTheMonolithApply = "BreakTheMonolith_Apply",
+  BreakTheMonolithStart = "BreakTheMonolith_Start",
```

Comment:
I talked with @PazYanoverr  and we agreed to use the prefix "ArchitectureRedesign_" instead of "BreakTheMonolith"

```suggestion
   ArchitectureRedesign_StartRedesign = " architectureRedesign_StartRedesign",
   ArchitectureRedesign_Apply = "architectureRedesign__Apply",
   ArchitectureRedesign_StartBreakTheMonolith = "architectureRedesign__StartBreakTheMonolith",
```